### PR TITLE
feat(resourceStrings): Strings object

### PR DESCRIPTION
**Pull request by nsbtr**
- Added resource files
- Added copy from landing page to start
- Samples of strings that require components as inputs

### CODE REVIEW

File: src/data/landing.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import React, { Fragment } from 'react';
+
+const strings = {
+  'button/apply': 'Apply Today',
+  'button/courseOverview': 'Course Overview',
+  'landing/first/title': Semibold => {
+    return (
+      <span>
+        At Saraswati Retreats, we believe your education should be{' '}
+        <Semibold>extraordinary.</Semibold>
```
Comment (nsbtr) [inline src/data/landing.js]: @johnathanfong Sample string with component as input

File: src/components/LandingModule.js [hunk 2]
```diff
@@ -15,18 +14,16 @@ const Semibold = styled.span`
...
       <HeroWithContent src={mainImage} width={50}>
-        <H1>
-          At Saraswati Retreats, we believe your education should be{" "}
-          <Semibold>extraordinary</Semibold>.
-        </H1>
-        <BodyText>
-          Learn to code your own website in the heart of Bali. Apply for our
-          14-day retreat today!
-        </BodyText>
+        <H1>{strings['landing/first/title'](Semibold)}</H1>
```
Comment (nsbtr) [inline src/components/LandingModule.js]: @johnathanfong Sample string with component as input

**Comment by colindamelio** [comment]
@johnathanfong To avoid conflicts and inability to keep working seamlessly, we should merge what you have and you can continue your refactoring with new PRs.

File: src/components/LandingModule.js (hunk 2 above)
Comment (JesseNoseworthy) [reply src/components/LandingModule.js]: If possible, a parent controls this so you can avoid passing down the entire `strings` object every time you need it.

File: src/routes/Home.js [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
+import Experience from 'components/Experience';
+import Information from 'components/Information';
+import Application from 'components/Application';
 
-const Home = () => {
+const Home = props => {
   return (
     <Fragment>
-      <LandingModule />
+      <LandingModule {...props} />
```
Comment (JesseNoseworthy) [inline src/routes/Home.js]: What is the benefit of passing _every_ prop down to this component? I know it makes things cleaner visually, but `LandingModule` arguably doesn't need every prop `Home` requires.

Comment (nsbtr) [reply src/routes/Home.js]: I just passed down the props so I could show a sample of the string use. You're right, we don't need to pass down every prop, but there's a lot of refactoring going on, and this `Landing Module` likely won't even exist very soon so I don't think it matters right now.

File: src/data/landing.js (hunk 1 above)
Comment (johnathanfong) [reply src/data/landing.js]: 💯

File: src/routes/Home.js (hunk 3 above)
Comment (johnathanfong) [reply src/routes/Home.js]: nope, it's gone lol. @nsbtr please update your PR by pulling down master

File: src/data/landing.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    );
+  },
+  'landing/overview/title': 'Front-End Fundamentals',
+  'landing/overview/description':
+    "With 10 days of in-class training, our goal is to ensure you have a strong foundation. Using HTML, CSS, and JAVASCRIPT, you will build your own website from scratch! No prior experience is required - just a desire to learn! It may be challenging, but we're excited to guide you through a comprehensive curriculum.",
+  'landing/experience/title': 'Providing you authentic Balinese experiences.',
+  'landing/experience/description':
+    'Without a doubt, Bali is beautiful. However, most people who visit the island miss an opportunity to discover authentic Balinese Culture. Saraswati Retreats strives to educate our guests beyond coding their website - experience local cuisine, traverse the landscape, and embrace what it means to be truly Balinese.',
+  'landing/accommodations/title': 'Your Accommodations',
+  'landing/accommodations/title':
```
Comment (johnathanfong) [inline src/data/landing.js]: should this be `landing/accommodations/description` ?

File: src/data/form.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  'form/firstName/label': 'First Name',
+  'form/lastName/label': 'Last Name',
+  'form/emailAddress/label': 'Email Address',
+  'form/phone/label': 'Phone Number (Please include Country code)',
+  'form/location/label': 'Where are you applying from? (City, Country)',
+  'form/selectRetreat/label': 'Select an upcoming retreat',
+  'form/accomplish/label':
+    'What would you like to accomplish during your Saraswati Experience?',
+};
```
Comment (johnathanfong) [inline src/data/form.js]: we'll need to add a list of available retreat dates

File: src/data/landing.js (hunk 4 above)
Comment (nsbtr) [reply src/data/landing.js]: yup, I'll fix it

**Pull request merged**